services/QuillLMS/app/services/dual_google_id_and_clever_id_resolver.rb

```diff
@@ -18,6 +18,10 @@ def run
     RESOLVERS.find { |resolver| send(resolver) }
   end
 
+  private def last_classroom
+    @last_classroom ||= user&.classrooms_i_teach&.sort_by(&:updated_at)&.last
```

Comment:
Would this be faster to use `ORDER BY DESC` since it basically lets us take `first`, and thus only pulls a single record from the database?  (I could be wrong, or it may not even matter much, but I figured I'd check.)
```ruby
user&.classrooms_i_teach&.order(updated_at: :desc).first
```

Obviously you'd have to change the stub logic in the spec, but I think the same idea would work with an `order` call.

Reply:
The `classrooms_i_teach` call returns an array so we can't use SQL at that point.

The previous `classrooms` call returned an ActiveRecord::Relation which is why it was originally a `order(:updated_at)`

Reply:
My guess is that is overwritten by this [one](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/models/concerns/teacher.rb#L45)

Reply:
Yeah.  That makes sense.  Also, that TODO is hilarious.  I guess we probably have both because there's some code somewhere that expects one form, and other code somewhere else that expects the other.